sdk/nodejs/automation/localWorkspace.ts

```diff
@@ -495,6 +495,17 @@ export class LocalWorkspace implements Workspace {
     async installPlugin(name: string, version: string, kind = "resource"): Promise<void> {
         await this.runPulumiCmd(["plugin", "install", kind, name, version]);
     }
+    /**
+     * Installs a plugin in the Workspace, from a third party server.
+     *
+     * @param name the name of the plugin.
+     * @param version the version of the plugin e.g. "v1.0.0".
+     * @param kind the kind of plugin, defaults to "resource"
+     * @param server the server to install the plugin from
+     */
+    async installPluginFromServer(name: string, version: string, kind = "resource", server: string): Promise<void> {
```

Comment:
Adding non default arguments are default arguments is bad, but I didn't want to break the API ordering here. Thoughts?